.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 jspm_packages
 .env
 dist
+config.js
+test-data.js
```

Comment:
depends. I think it's more likely to be inside a fixture file in the tests folder. What were you doing with it?